Source/WarpX.H

```diff
@@ -1489,6 +1484,18 @@ private:
     int noy_fft = 16;
     int noz_fft = 16;
 
+
+
+    //! If true, particles will be sorted in the order x -> y -> z -> ppc for faster deposition
+#if defined(AMREX_USE_CUDA)
+    bool sort_particles_for_deposition = true;
+#else
+    bool sort_particles_for_deposition = false;
+#endif
+
+    //! Specifies the type of grid used for the above sorting, i.e. cell-centered, nodal, or mixed
+    amrex::IntVect sort_idx_type = amrex::IntVect(AMREX_D_DECL(0,0,0));
+
```

Comment:
Should we prefix these private variables with `m_` since they are still members of a class?

Reply:
Done, @EZoni !